server.js

```diff
@@ -105,6 +105,38 @@ app.delete('/api/v1/players/:id', (request, response) => {
     })
 })
 
+app.post('/api/v1/players', (request, response) => {
+    const { body } = request;
+
+    const parameters = ['name', 'team', 'games_played', 'points_per_game', 'field_goal_percentage', 'three_point_percentage', 'free_throw_percentage', 'rebounds_per_game', 'assists_per_game', 'steals_per_game', 'blocks_per_game']; 
+
+  for (let requiredParameter of parameters) {
+    if (!body[requiredParameter]) {
+      return response
+              .status(422)
+              .json({error: `You\'re missing ${requiredParameter} from the expected format.`})
+    }
+  }
+  database('players').insert(body, 'id')
+    .then(body => response.status(201).json({ id: body[0] }))
+    .catch(error => response.status(500).json({ error }))
+}); 
+
+app.patch('/api/v1/players/:id', (request, response) => {
+  const id = parseInt(request.params.id); 
+  const { body } = request; 
+
+  database('players').where('id', id).update(body, 'id')
```

Comment:
this looks good. passing the id here will allow us to use it in the .then(). Need to add this in my code also